<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraAxis</class>
 <widget class="QMainWindow" name="CameraAxis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CameraAxis</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>461</width>
      <height>351</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="record">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>460</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="play">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>460</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fichier">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>File:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textFile">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>330</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Password">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>250</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Login">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Login:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IP">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>@IP:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textPassword">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textLogin">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textIP">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>170</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Format">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
